steps/step-fauxpasapp/0.1.0/step.yml

```diff
@@ -0,0 +1,115 @@
+title: Fauxpasapp Step
+summary: Step for Fauxpasapp
+description: |-
+  This is a Step for Fauxpasapp.
+
+  [Faux Pas](http://fauxpasapp.com)
+
+  This is not an officially supported plugin
+website: https://github.com/mackoj/step-fauxpasapp
+source_code_url: https://github.com/mackoj/step-fauxpasapp
+support_url: https://github.com/mackoj/step-fauxpasapp/issues
+published_at: 2016-03-18T16:40:04.503789528+01:00
+source:
+  git: https://github.com/mackoj/step-fauxpasapp.git
+  commit: 30e3f992f31c037db48ac65c0b019cca03e1a38e
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- script
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- FAUXPAS_LICENSE_TYPE: personal
+  opts:
+    is_dont_change_value: true
+    is_expand: false
```

Comment:
Most of the times `is_expand: true` is better then false, becouse it allows you to specify the input value with env var, otherwise just direct values are allowed.

`is_dont_change_value: true` means, this input is not editable on bitrise.io's graphical editor.